collect_app/release/output-metadata.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Was this added to the commit by mistake?

Reply:
Oh, yes I'm going to remove it.